app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
```

Comment:
- [ ] You forgot to set up the associations between `user and likes, comment, post` you should do that for example the user can have many likes, comments, and posts so it should the code like this :
```RUBY
has_many :posts, foreign_key: 'author_id'
```